src/main/java/org/sagebionetworks/web/client/widget/accessrequirements/TermsOfUseAccessRequirementWidget.java

```diff
@@ -99,6 +100,7 @@ public void setDataAccessSubmissionStatus(BasicAccessRequirementStatus status) {
 		// set up view based on DataAccessSubmission state
 		if (status.getIsApproved()) {
 			view.showApprovedHeading();
+			view.setAccessRequirementIDVisible();
```

Comment:
Visibility should be determined based on ACT member status, not approval status. See [ManagedACTAccessRequirementWidget](https://github.com/Sage-Bionetworks/SynapseWebClient/blob/5351f7427b88b8752f54ae598a530ec6a18f1d4e/src/main/java/org/sagebionetworks/web/client/widget/accessrequirements/ManagedACTAccessRequirementWidget.java#L123)